python/tskit/tables.py

```diff
@@ -2080,7 +2080,7 @@ def copy(self):
         Returns a deep copy of this TableCollection.
 
         :return: A deep copy of this TableCollection.
-        :rtype: .TableCollection
```

Comment:
This already works, doesn't it?

Reply:
I'm pretty sure I changed that because CircleCI was complaining.

Reply:
It's not throwing any errors for me, and I think we're trying to standardise on bare type names for these rtypes rather than sphinx markup. It would be better to back out of this change I think.